ProjektAuto/Program.cs

```diff
@@ -0,0 +1,3 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
+string promenna = ""
```

Comment:
Tady ti chybi strednik
```suggestion
string promenna = "";
```